networkit/centrality.pyx

```diff
@@ -2928,3 +2932,120 @@ class SciPyPageRank(SpectralCentrality):
 
 		self.eigenvector = spectrum[1][0]
 		self.eigenvalue = spectrum[0][0]
+
+##### from branch ComplexPaths (K. Ahrens)
+
+cdef extern from "<networkit/centrality/ComplexPaths.hpp>" namespace "NetworKit::ComplexPathAlgorithm":
+
+	cpdef enum _Mode  "NetworKit::ComplexPathAlgorithm::Mode":
+		singleNode,
+		allNodes
+
+cdef class ComplexPathMode(object):
+	singleNode = _Mode.singleNode
+	allNodes   = _Mode.allNodes
+
+cdef extern from "<networkit/centrality/ComplexPaths.hpp>":
+	cdef cppclass _ComplexPaths "NetworKit::ComplexPathAlgorithm"(_Algorithm):
+		_ComplexPaths(_Graph G, count threshold, _Mode mode, node start) except +
+		vector[double] getPLci() except +
+		void normalize() except +
+		vector[node] getAdopters() except +
+		_Graph &getComplexGraph() except +
+
+cdef class ComplexPaths(Algorithm):
+	""" 
+	ComplexPaths(G, threshold, mode, start)
+
+	according to:
+	[ Guilbeault, D., Centola, D. Topological measures for 
+	  identifying and predicting the spread of complex contagions. 
+	  Nat Commun 12, 4430 (2021). 
+	  https://doi.org/10.1038/s41467-021-24704-6 ]
+ 
+	Parameters
+	----------
+	G : networkit.Graph
+		The graph.
+
+	threshold : int
+		The complex path width: minimal number of neighbors
+
+	mode : ComplexPathMode
```

Comment:
Writing the parameter this way will likely break the rendering of the doc. Take a look at other examples in NetworKit at how this is written - for example `Diameter`:  https://github.com/networkit/networkit/blob/master/networkit/distance.pyx#L487

Also, the overview over included enum-parameters should be added above the parameter section.